src/style.css

```diff
@@ -0,0 +1,126 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 800;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+}
+
+.search-form {
+  position: relative;
+}
+
+.search-form_big {
+  margin: 20px auto;
+}
+
+.search-form__input {
+  font-family: "Avenir", sans-serif;
+  outline: none;
+  font-weight: 300;
+}
+
+.search-form__input-big {
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+  width: 100%;
+  font-weight: 300;
+  height: 70px;
+  padding: 0 0 0 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search-form__input-small {
+  width: 100%;
+  height: 42px;
+  padding-left: 33px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 19px;
+  color: #3d4e61;
+}
+
+.search-form__input-big::placeholder {
+  color: #3d4e61;
+}
+
+.search-form__input-small::placeholder {
+  color: #3d4e61;
+}
+
+.search-form__input-big:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search-form__input-big:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: bold;
+}
+
+.search-form__input-big:focus::placeholder {
+  font-weight: bold;
+}
+
+.search-form__input-small:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search-form__input-small:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: bold;
+}
+
+.search-form__input-small:focus::placeholder {
+  font-weight: bold;
+}
+
+.search-form__label::after {
+  position: absolute;
+  content: "";
+  display: block;
+}
+
+.search-form__label-big::after {
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.search-form__label-small:after {
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
```

Comment:
https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md
recheck 2nd point and apply it everywhere